cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesBoundary.java

```diff
@@ -65,10 +68,28 @@ public PowerFlow powerFlowAtNode(String node) {
         return nodesPowerFlow.get(node);
     }
 
-    public void addEquipmentAtNode(PropertyBag line, String node) {
```

Comment:
This method was public so please, deprecate it

Reply:
Done